DSharpPlus.Interactivity/Extensions/MessageExtensions.cs

```diff
@@ -62,6 +63,14 @@ public static Task<InteractivityResult<DiscordMessage>> GetNextMessageAsync(this
         public static Task<InteractivityResult<ComponentInteractionCreateEventArgs>> WaitForButtonAsync(this DiscordMessage message, TimeSpan? timeoutOverride = null)
             => GetInteractivity(message).WaitForButtonAsync(message, timeoutOverride);
 
+        /// <summary>
+        /// Waits for any button to be pressed on the specified message.
+        /// </summary>
+        /// <param name="message">The message to wait on.</param>
+        /// <param name="token">A custom cancellation token that can be cancelled at any point.</param>
+        public static Task<InteractivityResult<ComponentInteractionCreateEventArgs>> WaitForButtonAsync(this DiscordMessage message, CancellationToken? token = null)
```

Comment:
Do not make CancellationToken nullable. make it take `default` instead. This applies everywhere else.